ReactWindows/ReactNative.Shared/Animated/TrackingAnimatedNode.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Portions derived from React Native:
+// Copyright (c) 2015-present, Facebook, Inc.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json.Linq;
+
+namespace ReactNative.Animated
+{
+
+    class TrackingAnimatedNode : AnimatedNode
+    {
+        private readonly NativeAnimatedNodesManager _manager;
+
+        private readonly int _animationId;
+        private readonly int _toValueNode;
+        private readonly int _valueNode;
+        private readonly JObject _animationConfig;
+
+        public TrackingAnimatedNode(int tag, JObject config, NativeAnimatedNodesManager manager)
+            : base(tag)
+        {
+            _manager = manager; 
+            _animationId = config.Value<int>("animationId");
+            _toValueNode = config.Value<int>("toValue");
+            _valueNode = config.Value<int>("value");
+
+            var animationConfig = config.Value<JObject>("animationConfig");
+            _animationConfig = (JObject)animationConfig.DeepClone();
+        }
+
+        public override void Update()
+        {
+            AnimatedNode toValue = _manager.GetNodeById(_toValueNode);
+            _animationConfig["toValue"] = (toValue as ValueAnimatedNode).Value;
```

Comment:
Probably not likely to happen (as there are likely checks in JS land), but in isolation this is apt to throw NullReferenceException.

Reply:
I.e., hard cast is fine.

Reply:
done

---
In reply to: [206289422](https://github.com/Microsoft/react-native-windows/pull/1918#discussion_r206289422) [](ancestors = 206289422)